Upstream: Not applicable (local configuration)
Reason: Sets config defaults for Alpine

--- a/conf/cupsd.conf.in
+++ b/conf/cupsd.conf.in
@@ -154,12 +154,12 @@
 
   # Job-related operations must be done by the owner or an administrator...
   <Limit Create-Job Print-Job Print-URI Validate-Job>
-    AuthType Negotiate
+    AuthType Default
     Order deny,allow
   </Limit>
 
   <Limit Send-Document Send-URI Hold-Job Release-Job Restart-Job Purge-Jobs Set-Job-Attributes Create-Job-Subscription Renew-Subscription Cancel-Subscription Get-Notifications Reprocess-Job Cancel-Current-Job Suspend-Current-Job Resume-Job Cancel-My-Jobs Close-Job CUPS-Move-Job CUPS-Get-Document>
-    AuthType Negotiate
+    AuthType Default
     Require user @OWNER @SYSTEM
     Order deny,allow
   </Limit>
@@ -180,7 +180,7 @@
 
   # Only the owner or an administrator can cancel or authenticate a job...
   <Limit Cancel-Job CUPS-Authenticate-Job>
-    AuthType Negotiate
+    AuthType Default
     Require user @OWNER @CUPS_DEFAULT_PRINTOPERATOR_AUTH@
     Order deny,allow
   </Limit>
